ja/faq/external-resources.md

```diff
@@ -6,10 +6,10 @@ description: Nuxt.js で外部リソースを使うには？
 ## グローバルな設定
 
 head オブジェクトもしくは関数の中に、外部のリソースを含めることができます。
-1つ目の例ではオブジェクトを使用し、2つ目の例では関数を使用します。
-computed プロパティもしくは data プロパティのような Vue コンポーネントから値を使用したい場合、 関数として `head` を使用して、最終的な head オブジェクトを返すこともできます。
+[head API docs](https://ja.nuxtjs.org/api/pages-head/) で説明されているように、次の例では `head` をオブジェクトと関数として使用します。
+computed プロパティもしくは data プロパティのような Vue コンポーネントから値を使用したい場合、`head()` メソッドを使用して、最終的な head オブジェクトを返すこともできます。
 
-`nuxt.config.js` 内でリソースをインクルードします:
+`nuxt.config.js` 内でリソースをインクルードします（head オブジェクトの中）:
```

Comment:
I thought it might be better to translate like this. Because this document explains both of `head` object and `head `function. What do you think?
```suggestion
`nuxt.config.js` 内でリソースをインクルードします（この例では head オブジェクト）:
```

Reply:
Thanks!! I agree with your opinion. This is better than my translation!